[2024-01-30T15:58:58.414+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sample_python_dag.test_task scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-01-30T15:58:58.440+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sample_python_dag.test_task scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-01-30T15:58:58.441+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-01-30T15:58:58.472+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): test_task> on 2024-01-29 00:00:00+00:00
[2024-01-30T15:58:58.485+0000] {standard_task_runner.py:60} INFO - Started process 911 to run task
[2024-01-30T15:58:58.502+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sample_python_dag', 'test_task', 'scheduled__2024-01-29T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/mon dag/squares_dags.py', '--cfg-path', '/tmp/tmpj8zbrkw1']
[2024-01-30T15:58:58.506+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask test_task
[2024-01-30T15:58:58.601+0000] {task_command.py:423} INFO - Running <TaskInstance: sample_python_dag.test_task scheduled__2024-01-29T00:00:00+00:00 [running]> on host d8d040d8064b
[2024-01-30T15:58:58.680+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ledatascientist@ds.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sample_python_dag' AIRFLOW_CTX_TASK_ID='test_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-29T00:00:00+00:00'
[2024-01-30T15:58:58.682+0000] {logging_mixin.py:188} INFO - ['/home/***/.local/bin', '/usr/local/lib/python38.zip', '/usr/local/lib/python3.8', '/usr/local/lib/python3.8/lib-dynload', '/home/***/.local/lib/python3.8/site-packages', '/usr/local/lib/python3.8/site-packages', '/opt/***/dags', '/opt/***/config', '/opt/***/plugins']
[2024-01-30T15:58:58.683+0000] {logging_mixin.py:188} INFO - Hello, these are 10 perfect squares
[2024-01-30T15:58:58.683+0000] {logging_mixin.py:188} INFO - 0
[2024-01-30T15:58:58.683+0000] {logging_mixin.py:188} INFO - 1
[2024-01-30T15:58:58.684+0000] {logging_mixin.py:188} INFO - 4
[2024-01-30T15:58:58.684+0000] {logging_mixin.py:188} INFO - 9
[2024-01-30T15:58:58.684+0000] {logging_mixin.py:188} INFO - 16
[2024-01-30T15:58:58.684+0000] {logging_mixin.py:188} INFO - 25
[2024-01-30T15:58:58.685+0000] {logging_mixin.py:188} INFO - 36
[2024-01-30T15:58:58.685+0000] {logging_mixin.py:188} INFO - 49
[2024-01-30T15:58:58.685+0000] {logging_mixin.py:188} INFO - 64
[2024-01-30T15:58:58.686+0000] {logging_mixin.py:188} INFO - 81
[2024-01-30T15:58:58.686+0000] {python.py:201} INFO - Done. Returned value was: Done
[2024-01-30T15:58:58.752+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sample_python_dag, task_id=test_task, execution_date=20240129T000000, start_date=20240130T155858, end_date=20240130T155858
[2024-01-30T15:58:58.848+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-30T15:58:58.873+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
